final/index.html

```diff
@@ -40,16 +40,14 @@ <h1 class="header__title">Weather PWA</h1>
           </div>
         </div>
         <div class="description">
-          <div class="feels-like">
-            <span class="value"></span><span class="scale">°F</span>
-          </div>
-          <div class="precip"></div>
```

Comment:
This deletion goes one line beyond the "feels-like" div, and also clobbers "precip". Intended?